ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/toolbars/Bundle.properties

```diff
@@ -0,0 +1,4 @@
+mainWindow.swing.firmware.toolbarTitle = Firmware selection toolbar
```

Comment:
What I meant was that I can not use LocalizingService to translate a title like this:
```java 
public final static String PauseTitle = Localization.getString("mainWindow.swing.pauseButton", lang);
```

And use it in the registration like this because it isn't a constant:
```java
@ActionRegistration(
        iconBase = PauseAction.ICON_BASE,
        displayName = LocalizingService.PauseTitle,
        lazy = false)
```

However, I've moved the key constant to the localizing service and reference that as bundle key. But this requires that there is a bundle file in the same directory.